packages/laconia-event/src/apigateway/parseRequestBody.js

```diff
@@ -13,8 +13,8 @@ const parseJsonBody = body => {
   }
 };
 
-module.exports = event => {
-  const contentType = event.headers["Content-Type"] || "";
+module.exports = (event, headers) => {
```

Comment:
It's a good change! This is internal function anyway, it shouldn't break any backward compatibility